src/apps/components/AppAlerts/SidebarAppAlert.test.tsx

```diff
@@ -0,0 +1,135 @@
+      hasManagedAppsPermission: true,
+    });
+    (useAppFailedPendingWebhooksLazyQuery as jest.Mock).mockReturnValue([
+      jest.fn(),
+      {
+        data: {
+          apps: {
+            edges: [
+              {
+                node: {
+                  webhooks: [
+                    {
+                      failedDelivers: { edges: [] },
+                      pendingDelivers: {
+                        edges: [
+                          {
+                            node: {
+                              attempts: {
+                                edges: [
+                                  {
+                                    node: {
+                                      status: "FAILED",
+                                      createdAt: "2023-01-19T09:50:43.343Z",
+                                    },
+                                  },
+                                ],
+                              },
+                            },
+                          },
+                        ],
+                      },
+                    },
+                    {
+                      failedDelivers: { edges: [] },
+                      pendingDelivers: null,
+                    },
+                  ],
+                },
+              },
+            ],
+          },
+        },
+      },
+    ]);
+
+    // Act
+    await act(async () => {
+      render(<SidebarAppAlert />);
+    });
+
+    const trigger = screen.getByTestId("sidebar-app-alert-trigger");
+
+    fireEvent.focus(trigger);
+
+    await screen.findByRole(/tooltip/);
+
+    // Assert
+    // Radix tooltip content mounts twice - https://github.com/radix-ui/primitives/issues/3034
+    expect(screen.queryAllByText("Issues found.{break}Review app alerts.")[0]).toBeInTheDocument();
+  });
+
+  it("doesn't display sidebar alert dot when there are no webhook failures", async () => {
+    // Arrange
+    (useHasManagedAppsPermission as jest.Mock).mockReturnValue({
+      hasManagedAppsPermission: true,
+    });
+    (useAppFailedPendingWebhooksLazyQuery as jest.Mock).mockReturnValue([
+      jest.fn(),
+      {
+        data: {
+          apps: {
+            edges: [
+              {
+                node: {
+                  webhooks: [
+                    {
+                      failedDelivers: { edges: [] },
+                      pendingDelivers: {
+                        edges: [],
+                      },
+                    },
+                    {
+                      failedDelivers: { edges: [] },
+                      pendingDelivers: null,
+                    },
+                  ],
+                },
+              },
+            ],
+          },
+        },
+      },
+    ]);
+
+    // Act
+    await act(async () => {
+      render(<SidebarAppAlert />);
+    });
+
+    // Assert
+    expect(screen.queryByTestId("sidebar-app-alert-trigger")).not.toBeInTheDocument();
```

Comment:
Next time please don't add test-id for component/unit tests, You should match element by something more semantic such as role, element name, text, some attribute etc.